src/visualization/types/Map/view.tsx

```diff
@@ -109,6 +109,28 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
     bingKey: '',
   }
 
+  let layersOpts = layers
```

Comment:
```suggestion
  let layersOpts = [...layers]
```

this will create a copy for you. the way it is now will create a reference, and changing `layerOpts` will change `layers`.

not a big deal thought since `layers` doesn't get used here, but good practice.

Reply:
yeah makes sense. good idea.